pandas/io/pytables.py

```diff
@@ -579,12 +582,8 @@ def open(self, mode='a', **kwargs):
         if self.is_open:
             self.close()
 
-        if self._complib is not None:
-            if self._complevel is None:
-                self._complevel = 9
-            self._filters = _tables().Filters(self._complevel,
-                                              self._complib,
-                                              fletcher32=self._fletcher32)
+        self._filters = _tables().Filters(self._complevel, self._complib,
```

Comment:
I don't think you can do this is complevel is == 0, this changes the semantics to add a filter that doesn't do compression. I don't know if this is the same.